tests/Fixer/Operator/IncrementStyleFixerTest.php

```diff
@@ -138,6 +138,36 @@ public function provideFixPreIncrementCases()
             ['<?php foo($a, ++$b);'],
             ['<?php $a[++$b];'],
             ['<?php echo ++$a;'],
+
+            [
+                '<?php class Test {
+    public function foo() : void {
```

Comment:
Don't these typehints need to go because of 5.6?

(Almost the same haha)

Reply:
PHP 5.6 doesn't want to be forgotten :(